cmd/lncli/invoicesrpc_active.go

```diff
@@ -35,45 +35,56 @@ func getInvoicesClient(ctx *cli.Context) (invoicesrpc.InvoicesClient, func()) {
 var settleInvoiceCommand = cli.Command{
 	Name:     "settleinvoice",
 	Category: "Invoices",
-	Usage:    "Reveal a preimage and use it to settle the corresponding invoice.",
-	Description: `
-	Todo.`,
-	ArgsUsage: "preimage",
+	Usage: "Reveal a preimage and use it to settle the corresponding " +
+		"invoice. In case lnd already knows the preimage, it is " +
+		"sufficient to only provide the payment hash.",
+	ArgsUsage: "[preimage]",
 	Flags: []cli.Flag{
+		cli.StringFlag{
+			Name: "hash",
```

Comment:
once pay addr index and set id index are  we can just pass the `set_id` here. `set_id` tho maybe isn't the best name for the rpc interface, maybe `payment_id`?